src/main/java/org/jitsi/videobridge/LipSyncHack.java

```diff
@@ -204,52 +284,40 @@ void onRTPTranslatorWillWriteAudio(
             if (acceptedVideoSSRCs.contains(receiveVideoSSRC)
                 || injections.containsKey(receiveVideoSSRC))
             {
-                // No need to hack this SSRC.
+                // No need to hack this SSRC, as it's either already accepted
```

Comment:
So, an SSRC is assigned the accepted flag when at least 1 RTP video media packet has been accepted (or forwarded if you want).